src/renderer/entities/governance/model/referendum.ts

```diff
@@ -1,74 +1,77 @@
 import { type ApiPromise } from '@polkadot/api';
 import { createEffect, createEvent, createStore, sample } from 'effector';
-import { interval, readonly } from 'patronum';
+import { readonly } from 'patronum';
 
 import { type Chain, type ChainId, type Referendum, type ReferendumId } from '@/shared/core';
-import { addUnique } from '@shared/lib/utils';
+import { merge, nonNullable } from '@shared/lib/utils';
 import { governanceService } from '../lib/governanceService';
-
-type RequestListParams = {
-  chain: Chain;
-  api: ApiPromise;
+import { governanceSubscribeService } from '../lib/governanceSubscribeService';
+import { createSubscriber } from '../utils/createSubscriber';
+
+const mergeReferendums = (a: Referendum[], b: Referendum[]) => {
```

Comment:
what do you think to move it so referendum or governance service?

Reply:
services holds mostly business logic, not transport-level mappings. I don't think that this is a good idea